app/components/folio/captcha/turnstile_component.js

```diff
@@ -0,0 +1,67 @@
+window.Folio.Stimulus.register('f-captcha-turnstile', class extends window.Stimulus.Controller {
+  static values = { siteKey: String }
+
+  connect() {
+    if (!document.querySelector("#cloudflare-turnstile-script")) {
+      this.loadTurnstileScript()
+    } else {
+      this.renderTurnstile()
+    }
+
+    document.addEventListener("turbolinks:load", this.renderTurnstile.bind(this))
+    document.addEventListener("turbolinks:before-render", this.removeTurnstile.bind(this))
```

Comment:
Na tohle mame https://github.com/sinfin/folio/blob/master/app/assets/javascripts/folio/remote_scripts.js

Reply:
```
window.Folio.RemoteScripts.run({
  key: 'turnstile',
  urls: ['https://challenges.cloudflare.com/turnstile/v0/api.js?render=explicit']
}, () => {
  this.renderTurnstile()
}, () => {
  console.error('error')
})
```

Reply:
Tohle to furt zustalo. Cely connect by mel s tema remote scripts byt jen `this.loadTurnstileScript()`